changelog/unreleased/fix-disable-download-activity.md

```diff
@@ -0,0 +1,6 @@
+Bugfix: Disable download activity
+
+We disable the public link download activity until we have a proper solution for it.
```

Comment:
"We disable the ~~public link~~ download activity ..."

The change is not only for public link downloads.